### PR TITLE
Bump log4j-over-slf4j to 1.7.33

### PR DESCRIPTION
**Pull request by LesTR**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -96,6 +96,7 @@
     <commons-io.version>2.7</commons-io.version>
     <license-maven-plugin.version>3.0</license-maven-plugin.version>
     <log4j.version>2.17.1</log4j.version>
+    <log4j.over.slf4j.version>1.7.33</log4j.over.slf4j.version>
```
Comment (je-ik) [inline pom.xml:99]: The convention used for other dependencies seems to be `log4j-over-slf4j.version`. Can we use that?

**Review by je-ik — COMMENTED** [review]
Nit: I think the naming convention is slightly different. We can merge it at will, though.

File: pom.xml (hunk 1 above)
Comment (LesTR) [reply pom.xml]: Changed.

**Review by je-ik — APPROVED** [review]
LGTM

**Pull request merged**